config/llmisvc/config-llm-decode-worker-data-parallel.yaml

```diff
@@ -72,7 +72,7 @@ spec:
               # Leader-only launch
               #################
               vllm serve \
-                {{ .Spec.Model.Name }} \
+                /mnt/models \
```

Comment:
we will probably need to set `--served-model-name`

Reply:
Agreed. I added the argument to all affected templates in 9f785d3.